index.html

```diff
@@ -26,6 +26,7 @@
         <header>My TodoList</header>
         <div id="formContainer">
             <form onsubmit="addTask(event)" id="tasksForm">
+
```

Comment:
this PR is useless.... i don't get to see the changes since everything is already in branch `main`